### PR TITLE
New google cloud broker

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- gcp service broker

**Related issue(s)**
https://github.com/kyma-project/addons/issues/94

### CODE REVIEW

File: milv.config.yaml [hunk 1]
```diff
@@ -8,3 +8,6 @@ files:
   - path: "./addons/addons/azure-service-broker-0.0.1/README.md"
     config:
       white-list-external: ["https://github.com/kyma-project/addons/blob/master/addons/azure-service-broker-0.0.1/docs/broker/plans-details.md", "https://github.com/kyma-project/addons/blob/master/addons/azure-service-broker-0.0.1/docs/broker/overview.md#create-a-secret"]
+  - path: "./addons/addons/google-cloud-platform-service-broker-0.1.0/docs/google-memorystore-redis/plans-details.md"
+    config:
+      white-list-external: ["[-0-9a-z]*[a-z0-9]$"]
```
Comment (magicmatatjahu) [inline milv.config.yaml]: Very sorry... `white-list-internal`

File: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: Overview
+type: Overview
+---
+
+>**NOTE:** To provision this class, first you must create a Secret. Read the following document to learn how.
+
+The Google Cloud Platform Service Broker class exposes services from the [Google Cloud Platform Service Broker](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker)
+in a given Namespace of the Kyma cluster.
+
+The embedded MySQL DB is used to keep the broker's state so uninstalling the broker will delete all information about created instances but created services will still exists.
```
Comment (klaudiagrz) [inline addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md]:
```suggestion
The embedded MySQL database is used to keep the broker's state. Uninstalling the broker will delete all information about created instances but created services will still exist.
```

File: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Create a Secret
+
+To add the Google Cloud Platform Service Broker to your Namespace, you must prepare a service account and a
```
Comment (klaudiagrz) [inline addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md]:
```suggestion
To add the Google Cloud Platform Service Broker to your Namespace, prepare a service account and a 
```

File: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+The embedded MySQL DB is used to keep the broker's state so uninstalling the broker will delete all information about created instances but created services will still exists.
+
+## Create a Secret
+
+To add the Google Cloud Platform Service Broker to your Namespace, you must prepare a service account and a 
+JSON access key. 
+
+Follow these steps to create a Kubernetes Secret which contains a JSON access key:
+1. Open https://console.cloud.google.com/ and select your project.
```
Comment (klaudiagrz) [inline addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md]:
```suggestion
1. Open `https://console.cloud.google.com/` and select your project.
```

File: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+## Service description
+
+The `GCP Service Broker` service provides the following plan names and descriptions:
+
+| Plan Name | Description |
+|-----------|-------------|
+| `Default` | Installs the Google Cloud Platform (GCP) Service Broker in a default configuration. |
+
+### Provisioning parameters
```
Comment (klaudiagrz) [inline addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md]:
```suggestion
## Provisioning parameters
```

File: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+These are the provisioning parameters for the given plans:
```
Comment (klaudiagrz) [inline addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md]:
```suggestion
These are the provisioning parameters for the `Default` plan:
```

File: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+#### Default plan
```
Comment (klaudiagrz) [inline addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md]:
```suggestion

```

**Pull request merged**